ttnn/cpp/ttnn/operations/bernoulli/bernoulli.cpp

```diff
@@ -0,0 +1,19 @@
+
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "bernoulli.hpp"
+
+#include "device/bernoulli_device_operation.hpp"
+
+namespace ttnn::operations::bernoulli {
+Tensor Bernoulli::invoke(
+    const Tensor& input,
+    const std::optional<Tensor>& out,
+    const std::optional<DataType>& out_dtype,
```

Comment:
It should be 'output.' I think it's a mistake on Torch's part for not having consistency. Let's make the change across the entire repo.

Reply:
I've updated the param.